### PR TITLE
new article about routing in react

### PR DESCRIPTION
**Pull request by caroleolivier**
Thanks Si!

### CODE REVIEW

File: _posts/2017-10-13-routing-in-react-app.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: Routing in a React application
```
Comment (caroleolivier) [inline _posts/2017-10-13-routing-in-react-app.md:3]: I am not sure about the title...

Comment (simonproffitt) [reply _posts/2017-10-13-routing-in-react-app.md:3]: I think it's fine - I didn't know what it meant at the start but did by the end :)

**Review by simonproffitt — APPROVED** [review]
I can't see any more grammatical errors and I found it easier to understand with that changes you've made. Ship it!

**Comment by caroleolivier** [comment]
Thanks a lot Si

**Pull request merged**